tests/kafkatest/services/kafka.py

```diff
@@ -54,6 +54,14 @@ def start(self):
                 topic_cfg["topic"] = topic
                 self.create_topic(topic_cfg)
 
+        # Sanity check - are the broker processes actually running?
```

Comment:
Just a side note, but it'd be really nice to have some utility code to help parallelize stuff like this.